articles/azure-maps/migrate-from-bing-maps-web-services.md

```diff
@@ -23,202 +23,202 @@ Both Azure and Bing Maps provide access to spatial APIs through REST web service
-* [POI category search](/rest/api/maps/search/getsearchpoicategory): Search for points of interests by category. For example; "restaurant".
+* [Free-form address geocoding]: Specify a single address string (like `"1 Microsoft way, Redmond, WA"`) and process the request immediately. This service is recommended if you need to geocode individual addresses quickly.
+* [fuzzy search]: This API combines address geocoding with point of interest search. This API takes in a free-form string that can be an address, place, landmark, point of interest, or point of interest category and process the request immediately. This API is recommended for applications where users can search for addresses or points of interest from the same textbox.
+* [POI search]: Search for points of interests by name. For example, `"starbucks"`.
+* [POI category search]: Search for points of interests by category. For example, "restaurant".
 
 ## Calculate routes and directions
 
-Azure Maps can be used to calculate routes and directions. Azure Maps has many of the same functionalities as the Bing Maps routing service, such as;
+Azure Maps can be used to calculate routes and directions. Azure Maps has many of the same functionalities as the Bing Maps routing service, such as:
 
 * arrival and departure times
 * real-time and predictive based traffic routes
-* different modes of transportation; driving, walking, truck
+* different modes of transportation, driving, walking, truck
 * waypoint order optimization (traveling salesmen)
 
 > [!NOTE]
 > Azure Maps requires all waypoints to be coordinates. Addresses will need to be geocoded first.
 
-The Azure Maps routing service provides the following APIs for calculating routes;
+The Azure Maps routing service provides the following APIs for calculating routes:
 
-* [Calculate route](/rest/api/maps/route/getroutedirections): Calculate a route and have the request processed immediately. This API supports both GET and POST requests. POST requests are recommended when specifying a large number of waypoints or when using lots of the route options to ensure that the URL request doesn’t become too long and cause issues.
-* [Batch route](/rest/api/maps/route/postroutedirectionsbatchpreview): Create a request containing up to 1,000 route request and have them processed over a period of time. All the data will be processed in parallel on the server and when completed the full result set can be downloaded.
+* [Calculate route]: Calculate a route and have the request processed immediately. This API supports both `GET` and `POST` requests. `POST` requests are recommended when specifying a large number of waypoints or when using lots of the route options to ensure that the URL request doesn’t become too long and cause issues.
+* [Batch route]: Create a request containing up to 1,000 route request and have them processed over a period of time. All the data is processed in parallel on the server and when completed the full result set can be downloaded.
```

Comment:
The links in lines 198 and 199 don't render properly.

Reply:
@ShannonLeavitt how are they rendering? I'm not certain what to fix. Is it just these two links that are not rendering correctly?

Reply:
In the upper-right corner of the article here in the diff view, select **View file** to see a GitHub rendering of the article. There you can see that the links are (or were) surrounded by square brackets. The problem seems to be that the link list at the bottom of this article is missing a critical colon. I've added a suggestion there.

<img width="163" alt="image" src="https://user-images.githubusercontent.com/47186198/228638056-aac6a63e-bc06-40c2-bf29-9d6ed39979cc.png">

Reply:
Thank you. This appears to be rendering correctly now.